components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/ThrottlingApiServiceImpl.java

```diff
@@ -907,6 +997,323 @@ public Response throttlingPoliciesCustomRuleIdDelete(String ruleId,  MessageCont
         return null;
     }
 
+    /**
+     * Export a Throttling Policy by the policy name with/without specifying the policy type
+     * If policy type is not specified first found throttling policy is returned
+     *
+     * @param policyId   UUID of the throttling policy to be exported(for future use)
+     * @param policyName Name of the policy to be exported
+     * @param type       type of the policy to be exported
+     * @param format     format of the policy details
+     * @return required throttling policy
```

Comment:
Add a meaningful phrase implying what is returned

Reply:
Fixed with 30d0ecb9c29c0024fa182ba597c8eda4c128c451